Prox/Prox/Utilities/AnalyticsUtilities.swift

```diff
@@ -61,11 +61,16 @@ public struct AnalyticsEvent {
     static let PARAM_ACTION       = "action"
     static let NUM_CARDS          = "num_cards"
     static let CARD_INDEX         = "card_index"
+    static let FOREGROUND         = "foreground"
```

Comment:
The value changed between this version and the previous, which could affect the existing data. Is this okay?